From dd345d00bb54cc17f100f8430cc8d62d20c16537 Mon Sep 17 00:00:00 2001
From: Milan Crha <mcrha@redhat.com>
Date: Thu, 11 Aug 2016 12:36:02 +0200
Subject: [PATCH 226/595] The "headers-collapsed" change not propagated into
 the settings

---
 web-extensions/e-dom-utils.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/web-extensions/e-dom-utils.c b/web-extensions/e-dom-utils.c
index f671f77cca..e64be02782 100644
--- a/web-extensions/e-dom-utils.c
+++ b/web-extensions/e-dom-utils.c
@@ -570,6 +570,7 @@ toggle_headers_visibility (WebKitDOMElement *button,
 {
 	WebKitDOMElement *short_headers = NULL, *full_headers = NULL;
 	WebKitDOMCSSStyleDeclaration *css_short = NULL, *css_full = NULL;
+	GSettings *settings;
 	gboolean expanded;
 	const gchar *path;
 	gchar *css_value;
@@ -606,6 +607,11 @@ toggle_headers_visibility (WebKitDOMElement *button,
 
 	webkit_dom_html_image_element_set_src (
 		WEBKIT_DOM_HTML_IMAGE_ELEMENT (button), path);
+
+	settings = e_util_ref_settings ("org.gnome.evolution.mail");
+	g_settings_set_boolean (settings, "headers-collapsed", expanded);
+	g_clear_object (&settings);
+
  clean:
 	g_clear_object (&short_headers);
 	g_clear_object (&css_short);
-- 
2.15.0

