From b1a99009fa305f18bb6669e5402b8a50b3a92a8d Mon Sep 17 00:00:00 2001
From: Tomas Popela <tpopela@redhat.com>
Date: Tue, 23 Aug 2016 10:05:32 +0200
Subject: [PATCH 300/595] Fix the tests expectations

---
 e-util/test-html-editor-units.c | 84 ++++++++++++++++++++---------------------
 1 file changed, 42 insertions(+), 42 deletions(-)

diff --git a/e-util/test-html-editor-units.c b/e-util/test-html-editor-units.c
index f166151068..f295a5cd07 100644
--- a/e-util/test-html-editor-units.c
+++ b/e-util/test-html-editor-units.c
@@ -2194,104 +2194,104 @@ test_undo_style (TestFixture *fixture)
 		"action:bold\n"
 		"type:bold\n"
 		"undo:save\n" /* 2 */
-		"undo:undo:4\n"
+		"undo:undo:5\n"
 		"undo:test:2\n"
-		"undo:redo:4\n"
+		"undo:redo:5\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:4\n"
+		"undo:undo:5\n"
 		"type:bold\n"
 		"seq:CSlsc\n"
 		"action:bold\n"
-		"undo:save:\n" /* 2 */
-		"undo:undo:4\n"
+		"undo:save\n" /* 2 */
+		"undo:undo:5\n"
 		"undo:test:2\n"
-		"undo:redo:4\n"
+		"undo:redo:5\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:4\n"
+		"undo:undo:5\n"
 
 		"action:italic\n"
 		"type:italic\n"
 		"undo:save\n" /* 2 */
-		"undo:undo:6\n"
+		"undo:undo:7\n"
 		"undo:test:2\n"
-		"undo:redo:6\n"
+		"undo:redo:7\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:6\n"
+		"undo:undo:7\n"
 		"type:italic\n"
 		"seq:CSlsc\n"
 		"action:italic\n"
-		"undo:save:\n" /* 2 */
-		"undo:undo:6\n"
+		"undo:save\n" /* 2 */
+		"undo:undo:7\n"
 		"undo:test:2\n"
-		"undo:redo:6\n"
+		"undo:redo:7\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:6\n"
+		"undo:undo:7\n"
 
 		"action:underline\n"
 		"type:underline\n"
 		"undo:save\n" /* 2 */
-		"undo:undo:9\n"
+		"undo:undo:10\n"
 		"undo:test:2\n"
-		"undo:redo:9\n"
+		"undo:redo:10\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:9\n"
-		"type:italic\n"
+		"undo:undo:10\n"
+		"type:underline\n"
 		"seq:CSlsc\n"
 		"action:underline\n"
-		"undo:save:\n" /* 2 */
-		"undo:undo:9\n"
+		"undo:save\n" /* 2 */
+		"undo:undo:10\n"
 		"undo:test:2\n"
-		"undo:redo:9\n"
+		"undo:redo:10\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:9\n"
+		"undo:undo:10\n"
 
 		"action:strikethrough\n"
 		"type:strikethrough\n"
 		"undo:save\n" /* 2 */
-		"undo:undo:13\n"
+		"undo:undo:14\n"
 		"undo:test:2\n"
-		"undo:redo:13\n"
+		"undo:redo:14\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:13\n"
+		"undo:undo:14\n"
 		"type:strikethrough\n"
 		"seq:CSlsc\n"
 		"action:strikethrough\n"
-		"undo:save:\n" /* 2 */
-		"undo:undo:13\n"
+		"undo:save\n" /* 2 */
+		"undo:undo:14\n"
 		"undo:test:2\n"
-		"undo:redo:13\n"
+		"undo:redo:14\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:13\n"
+		"undo:undo:14\n"
 
 		"action:monospaced\n"
 		"type:monospaced\n"
 		"undo:save\n" /* 2 */
-		"undo:undo:10\n"
+		"undo:undo:11\n"
 		"undo:test:2\n"
-		"undo:redo:10\n"
+		"undo:redo:11\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:10\n"
+		"undo:undo:11\n"
 		"type:monospaced\n"
 		"seq:CSlsc\n"
 		"action:monospaced\n"
-		"undo:save:\n" /* 2 */
-		"undo:undo:10\n"
+		"undo:save\n" /* 2 */
+		"undo:undo:11\n"
 		"undo:test:2\n"
-		"undo:redo:10\n"
+		"undo:redo:11\n"
 		"undo:test\n"
 		"undo:drop\n" /* drop the save 2 */
-		"undo:undo:10\n",
-		HTML_PREFIX "<p>The first paragraph</p><p><br></p>" HTML_SUFFIX,
-		"The first paragraph\n"))
+		"undo:undo:11\n",
+		HTML_PREFIX "<p>The first paragraph text</p><p><br></p>" HTML_SUFFIX,
+		"The first paragraph text\n"))
 		g_test_fail ();
 }
 
@@ -2315,7 +2315,7 @@ test_undo_justify (TestFixture *fixture)
 		"type:left\n"
 		"seq:CSlsc\n"
 		"action:justify-left\n"
-		"undo:save:\n" /* 2 */
+		"undo:save\n" /* 2 */
 		"undo:undo:4\n"
 		"undo:test:2\n"
 		"undo:redo:4\n"
@@ -2335,7 +2335,7 @@ test_undo_justify (TestFixture *fixture)
 		"type:center\n"
 		"seq:CSlsc\n"
 		"action:justify-center\n"
-		"undo:save:\n" /* 2 */
+		"undo:save\n" /* 2 */
 		"undo:undo:6\n"
 		"undo:test:2\n"
 		"undo:redo:6\n"
@@ -2355,7 +2355,7 @@ test_undo_justify (TestFixture *fixture)
 		"type:right\n"
 		"seq:CSlsc\n"
 		"action:justify-right\n"
-		"undo:save:\n" /* 2 */
+		"undo:save\n" /* 2 */
 		"undo:undo:5\n"
 		"undo:test:2\n"
 		"undo:redo:5\n"
@@ -2388,7 +2388,7 @@ test_undo_indent (TestFixture *fixture)
 		"type:text\n"
 		"seq:CSlsc\n"
 		"action:indent\n"
-		"undo:save:\n" /* 2 */
+		"undo:save\n" /* 2 */
 		"undo:undo:5\n"
 		"undo:test:2\n"
 		"undo:redo:5\n"
-- 
2.15.0

